docs/configuration/large_language_model.md

```diff
@@ -88,21 +88,66 @@ ludwig train ...
 
 ## Input Features
 
-Currently, the LLM model type only supports a single input feature of type `text`.
+Because of the way LLMs work, they accept exactly one input, which is some amount of `text`. 
+This input can be built up from one or more columns of the input dataset and optional 
+interspersed static text. The following two examples illustrate the range of possibilities.
 
-If no `prompt` template is provided, this feature must correspond to a column
-in the input dataset. If a prompt template is provided, the rendered prompt
-will be used as the input feature value during training and inference.
+### Single Dataset Column
+
+If it is intended for the input to the LLM to be just the content of a single dataset column (without 
+any other prefixed or suffixed text), no `prompt` template should be provided and the `name` of the 
+feature must correspond to that specific dataset column. See the following example.
```

Comment:
Maybe we can also mention that it is still possible to wrap the single column with a static prompt template if desired.

Reply:
Please see my [last comment below](https://github.com/ludwig-ai/ludwig-docs/pull/356#issuecomment-2030970301)